controller/invitation.go

```diff
@@ -81,6 +81,40 @@ func (c *InvitationController) CreateInvite(ctx *app.CreateInviteInvitationConte
 	return ctx.Created()
 }
 
+// RescindInvite runs the revoke action.
+func (c *InvitationController) RescindInvite(ctx *app.RescindInviteInvitationContext) error {
+	currentIdentity, err := login.LoadContextIdentityIfNotDeprovisioned(ctx, c.app)
+	if err != nil {
+		return jsonapi.JSONErrorResponse(ctx, err)
+	}
+
+	invitationID, err := uuid.FromString(ctx.InviteTo)
+	if err != nil {
+		log.Error(ctx, map[string]interface{}{
+			"err":          err,
+			"invitationID": invitationID,
+		}, "failed to rescind invitation, invalid invitation id")
```

Comment:
After logging the error, do a `return jsonapi.JSONErrorResponse(ctx,errors.NewNotFoundError(..))`, because there's no need to go further.